src/index.ts

```diff
@@ -0,0 +1,15 @@
+export class Challenge {
+    buzz (numero : number) {
+        const result = new Promise(function(resolve, reject){
```

Comment:
This variable declaration is useless since you can directly return the new Promise:
```
return new Promise(function(resolve, reject)
{
   // code
});
```